### PR TITLE
Fix/dependency material ui picker

### PR DESCRIPTION
**Pull request by OscillatingMonkey**
FIX: 

material-ui-picker dependency did not resolve correctly

    RECREATE:

    1. Run a fresh npm build on latest react tools without adjusting any of the
    versions in the package files.

    2. Start the application. When you attempt to load a page the build will
    break saying that it cannot resolve MuiPickersUtilsProvider.

    CHANGES:

    Updated matierial-ui-picker dependency and changed the import to grab
    the provider from the root module in the package.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,3 @@
-[![Build Status](https://travis-ci.org/entaxy-project/entaxy.svg?branch=master)](https://travis-ci.org/entaxy-project/entaxy)
```
Comment (JackNeto) [inline README.md]: Why remove these?

File: package.json [hunk 2]
```diff
@@ -87,7 +87,7 @@
     "jest-enzyme": "^7.0.0",
     "jsdom": "^13.0.0",
     "lodash": "^4.17.5",
-    "material-ui-pickers": "^1.0.0-rc.17",
+    "material-ui-pickers": "^1.0.1",
```
Comment (JackNeto) [inline package.json]: 👍

File: package.json [hunk 3]
```diff
@@ -142,14 +141,6 @@
       "!src/store/**/types.js"
     ],
     "collectCoverage": true,
-    "coverageThreshold": {
```
Comment (JackNeto) [inline package.json]: These will keep us honest

File: README.md (hunk 1 above)
Comment (OscillatingMonkey) [reply README.md]: @kamalsaleh2

File: package.json (hunk 3 above)
Comment (OscillatingMonkey) [reply package.json]: @kamalsaleh2

**Comment by JackNeto** [comment]
Removed dependency on `material ui picker`

**Pull request merged**